_build/stac.hbs

```diff
@@ -38,5 +42,11 @@
       "rel": "license"
     }
   ],
-  "cube:dimensions": {{{copyJson CubeDimensions}}}

```

Comment:
Not sure if it's 100% correctly formatted, but we should add the thumbnail here as discussed. https://git.sinergise.com/team-6/openeo-platform/-/issues/11#note_250920

```
"assets": [
   "thumbnail": {
      "href": "https://collections.eurodatacube.com/{{Image}}",
   }
],
```

Reply:
Looks good, thanks.